### PR TITLE
Version 0.7

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#36 
#### What does this implement/fix? Explain your changes.
- Graph Line format added
- Menu optimized

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR ;-)
If everything is okay, please merge it after approval.

File: pyrgg/pyrgg.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ def input_filter(input_dict):
         filtered_dict["self_loop"] = 1
     if filtered_dict["multigraph"] not in [1, 2]:
         filtered_dict["multigraph"] = 1
-    if filtered_dict["output_format"] not in list(range(1, 12)):
+    if filtered_dict["output_format"] not in list(range(1, 13)):
```
Comment (sadrasabouri) [inline pyrgg/pyrgg.py:211]: Can't we use `len(SUFFIX_MENU)` here instead of 13?

File: pyrgg/params.py [hunk 2]
```diff
@@ -3,20 +3,20 @@
...
+
+MENU_ITEMS2 = {"vertices": "- Vertices Number : ",
+               "max_weight": "- Max Weight : ",
+               "min_weight": "- Min Weight : ",
+               "min_edge": "- Min Edge Number :",
+               "max_edge": "- Max Edge Number :",
+               "sign": "- Signed[1] or Unsigned[2]",
+               "direct": "- Directed[1] or Undirected[2]",
+               "self_loop": "- Self Loop[1] or No Self Loop[2]",
+               "multigraph": "- Simple[1] or Multigraph[2]"}
```
Comment (sadrasabouri) [inline pyrgg/params.py:19]: It seems better if we update CLI snapshots - provided in `README.md` - for release too.

File: pyrgg/pyrgg.py (hunk 1 above)
Comment (sepandhaghighi) [reply pyrgg/pyrgg.py]: You are right ;-)
Solved in https://github.com/sepandhaghighi/pyrgg/pull/38/commits/dee980ba42c8ce714b9864e740f0f56a1447cb0e

File: pyrgg/params.py (hunk 2 above)
Comment (sepandhaghighi) [reply pyrgg/params.py:19]: You are right about snapshots.
If you are okay, please work on it in another PR ;-)

File: pyrgg/pyrgg.py (hunk 1 above)
Comment (sadrasabouri) [reply pyrgg/pyrgg.py]: Ok, I'll handle it in #39 .

**Pull request merged**